disasm-test/Main.hs

```diff
@@ -492,16 +497,21 @@ parseBC pfx bc = do
 -- clang-generated bitcode will contain version-current output which might have
 -- newer elements and ordering.
 --
--- The compilerCube uses .c files as the input, and .ll files for the expected
--- output.  The assemblyCube uses the .ll file as both input and output.  The
--- actual testing done is very similar, and the assemblyCube always generates a
--- superset of the compilerCube tests (i.e. when no .c file is present).
+-- The cCompilerCube uses .c (C source) files as the input and .ll files for the
+-- expected output.  The ccCompilerCube uses .cc (C++ source) files as the input
+-- and .ll files for the expected output.  The assemblyCube uses the .ll file as
+-- both input and output.  The actual testing done is very similar, and the
+-- assemblyCube always generates a superset of the compilerCube tests (i.e. when
+-- no .c or .cc file is present).
 
-compilerCube :: TS.CUBE
-compilerCube = assemblyCube
-               { TS.rootName = "*.c"
-               , TS.sweetAdjuster = rangeMatch
-               }
+cCompilerCube :: TS.CUBE
+cCompilerCube = assemblyCube
+                { TS.rootName = "*.c"
+                , TS.sweetAdjuster = rangeMatch
+                }
+
+ccCompilerCube :: TS.CUBE
+ccCompilerCube = cCompilerCube { TS.rootName = "*.cc"}
```

Comment:
Ah, but you're specifically checking against `*.cc` here... I'm not sure if `tasty-sugar` easily supports having multiple file extensions in the `rootName`. If not, feel free to ignore this suggestion.